helmcharts/services/alloy/README.md

```
@@ -0,0 +1,369 @@
+If `alloy.configMap.content` is not provided, a [default configuration file][default-config] is
+used. When provided, `alloy.configMap.content` must hold a valid Alloy configuration file.
+
+[default-config]: ./config/example.alloy
+
+### alloy.securityContext
+
+`alloy.securityContext` sets the securityContext passed to the Grafana
+Alloy container.
+
+By default, Grafana Alloy containers are not able to collect telemetry from the
+host node or other specific types of privileged telemetry data. See [Collecting
```

markdownlint-cli2 (0.20.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "#collecting-logs-from-other-containers"

(MD052, reference-links-images)